packages/components/src/index.ts

```diff
@@ -1281,7 +1280,6 @@ export {
     SVGIcon,
     // general components
     Alert,
-    CollapsiblePanel,
```

Comment:
it looks like there are 2 usages of this in the provenance module. maybe this component should just move there? thoughts?

Reply:
nvm, just saw the PR for provenance!